### DISCUSSION TITLE
Welcome to osTicket Discussions!

### DISCUSSION THREAD

**Discussion by protich** [osTicket/osTicket#5742] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ntozier** [comment]
Hello all,
ntozier here forum moderator for the osTicket Forums and long time osTicket site admin for my employer.  You've all probably seen my posts on the forums and here on github.

**Comment by Richiricheh** [comment]
Hey,

Rich_C here. Been around a while.

Stay safe.

**Comment by caheba** [comment]
Hi,
thank you for your work on this project. Is this the right place to ask for changes / adds?

**Reply by caheba** [reply]
One point, I want to ask for is: ticket status "waiting":
Ticket does not show red or green on the list (maybe blue).
Another one: "accounting":
Instead of closing a ticket when work is finished, users forward it to accounting, ticket will be billed by accounting team, and then closed finally.

**Reply by ntozier** [reply]
If you want to make a change request you would open an issue report on the main repo and prefix the title with [Feature Request].

That being said you can add any ticket statuses that you want with out making a feature request.  Go to **Admin panel->manage->Lists->Ticket Statuses**

**Comment by snehasispy** [comment]
can anyone tell me which framework is used to build osticket .

**Reply by ntozier** [reply]
It's not.  osTicket is older than most frameworks.
